pkgs/mime/README.md

```diff
@@ -57,3 +57,14 @@ request
     .map((part) => part.fold(0, (p, d) => p + d))
     .listen((length) => print('Part with length $length'));
 ```
+
+## Determining file extension by MIME type
+
+The top level function `extensionFromMime` can be used to determine the
+file extension of a given MIME type.
+
+```dart
+print(extensionFromMime('text/html'));  // Will print html
```

Comment:
Conside formatting the comment as:
```dart
..)); // Prints "html".
```
(Dittos below.)

Be direct in comments. It saves words and usually avoids ambiguity. Present tense verbs win!
Distinguish different kinds of text. Here "html" is an actual string value, but is not distinguished from the other words.

Reply:
done